app/code/Magento/Checkout/CustomerData/DefaultItem.php

```diff
@@ -92,7 +92,7 @@ protected function doGetItemData()
             'configure_url' => $this->getConfigureUrl(),
             'is_visible_in_site_visibility' => $this->item->getProduct()->isVisibleInSiteVisibility(),
             'product_id' => $this->item->getProduct()->getId(),
-            'product_name' => $productName,
+            'productNameUnsanitizedHtml' => $productName,
```

Comment:
I think we need to bring back product_name, probably would be better to have it not escaped. What do you thing ?

Reply:
yes i will revert that